commands/root.go

```diff
@@ -5,10 +5,10 @@ import (
 
 	"github.com/spf13/cobra"
 
-	"github.com/git-bug/git-bug/commands/bridge"
-	"github.com/git-bug/git-bug/commands/bug"
+	bridgecmd "github.com/git-bug/git-bug/commands/bridge"
+	bugcmd "github.com/git-bug/git-bug/commands/bug"
 	"github.com/git-bug/git-bug/commands/execenv"
-	"github.com/git-bug/git-bug/commands/user"
+	usercmd "github.com/git-bug/git-bug/commands/user"
```

Comment:
this is showing up as a diff because it was removed from HEAD in a commit that's newer than your tree. you should be able to get rid of this automagically by rebasing on top of HEAD.